cms/migrations/0001_initial.py

```diff
@@ -3,13 +3,16 @@
 
 from cms.models import ACCESS_CHOICES, Page
 from cms.utils.conf import get_cms_setting
+from cms.constants import TEMPLATE_INHERITANCE_MAGIC
 from django.conf import settings
 from django.db import models, migrations
 import django.utils.timezone
 from django.utils.translation import ugettext_lazy as _
 
 template_choices = [(x, _(y)) for x, y in get_cms_setting('TEMPLATES')]
 
+TEMPLATE_DEFAULT = TEMPLATE_INHERITANCE_MAGIC if get_cms_setting('TEMPLATE_INHERITANCE') else get_cms_setting('TEMPLATES')[0][0]
```

Comment:
To keep consistency with the variable above, please lowercase this one. Or maybe uppercase both since they're both constants.